tests/VCR/Util/HttpUtilTest.php

```diff
@@ -128,4 +128,16 @@ public function testParseHeadersMultiple()
         $outputArray = HttpUtil::parseHeaders($inputArray);
         $this->assertEquals($excpetedHeaders, $outputArray);
     }
+
+    public function testParseHeadersIncludingColons()
```

Comment:
Awesome, I'll ask you to wait a bit before merge, we need to merge #166 before, after we do I'll ask you to update this PR this way me make sure everything is passing and then I'll merge yours, it should happen tomorrow!

Thanks a lot!